nuopc.runconfig

```diff
@@ -278,19 +278,19 @@ MED_attributes::
 ::
 
 CLOCK_attributes::
-     atm_cpl_dt = 99999     #not used
+     atm_cpl_dt = 99999 #not used
      calendar = GREGORIAN
      end_restart = .false.
      glc_avg_period = yearly
      glc_cpl_dt = 86400
```

Comment:
`glc_cpl_dt = 99999 #not used` instead of 86400

Reply:
updated in this commit adaf65c